newrelic-agent/src/main/java/com/newrelic/api/agent/NewRelicApiImplementation.java

```diff
@@ -98,11 +108,23 @@ public void noticeError(String message) {
 
     @Override
     public void noticeError(Throwable throwable, Map<String, ?> params, boolean expected) {
+        noticeError(throwable, params, expected, true);
+
+    }
+
+    @Override
+    public void noticeError(Throwable throwable, boolean expected) {
+        Map<String, String> params = Collections.emptyMap();
+        noticeError(throwable, params, expected, true);
+    }
+
+    private void noticeError(Throwable throwable, Map<String, ?> params, boolean expected, boolean expectedByApi) {
         try {
             ServiceFactory.getRPMService().getErrorService().reportException(throwable, filterErrorAtts(params, attributeSender), expected);
 
             MetricNames.recordApiSupportabilityMetric(MetricNames.SUPPORTABILITY_API_NOTICE_ERROR);
-            if (expected) {
+
+            if (expected && expectedByApi) {
```

Comment:
Ah, this should probably just be 
if (expectedByApi)

Reply:
That misses my point, I think. What happens if only `expected` evaluates to `true`? That is a case where a metric should be recorded, no?

Reply:
This metric should only be recorded if it is marked expected by the API. In my mind it is part of the issue that is reported.  Previously we were over reporting expected by API calls and not reporting consulting config (though in fact because of how we report supportability metrics for config it was getting recorded as such.